src/categoriesdialog.cpp

```diff
@@ -211,22 +210,22 @@ void CategoriesDialog::fillTable()
 
   int row = 0;
   for (std::vector<CategoryFactory::Category>::const_iterator iter =
```

Comment:
For some reason when I change this loop, it's failing to load the Category data causing the Table to be blank when you initialize the dialog.

I'm probably doing something wrong somewhere, but I'm not sure what.

Reply:
There is a `++row` in this for loop so you need to move it somewhere if you change to a range-loop.

Reply:
Yes, I did that. For some reason the element seemed to have uninitialized values when using the C++11 loop. I wasn't able to figure why the data wasn't loading properly.

Reply:
Okay, don't code when it's pushing 4AM. I was putting the ++row at the front of the loop instead of the back.

I'll push the updated version shortly.